### ISSUE TITLE
extern(System) in static bindings on Windows 32bit causes linker errors

### ISSUE THREAD

**Issue by MrSmith33** [DerelictOrg/DerelictGLFW3#12]
inside `derelict.glfw3.statfun` all the functions are marked as `extern(System)`.

This causes multiple errors like:
```
main.obj : error LNK2019: unresolved external symbol _glfwInit@0 referenced in function __Dmain
```

However, `extern(System)` does work when compiled in 64bit mode.

Then, I tested `extern(C)`, and on both 32 and 64 bit compilations it links.

Is that a problem with my static libs (I compiled them myself with MSVC 2013 to avoid additional dependencies), or `extern(System)` needs to be replaced by `extern(C)` on Windows?

**Comment by mdparker** [comment]
This is a bug. The functions are supposed to be extern(C). I'll fix it soon.

**Comment by MrSmith33** [comment]
Thanks!

**Comment by mdparker** [comment]
Finally got around to this. It's fixed now in 3.1.3 and 4.0.0-alpha.6.
